From 495a3a0599b340c5a6ebd56cd8c62429ad8dfedb Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 26 May 2025 08:23:22 +0300
Subject: [PATCH] [SW-229990] scripts: serdes: add config names

For making the future updates of serdes config easier to the editor, add
the name of each configuration.

Signed-off-by: Guy Nisan <gnisan@habana.ai>
Change-Id: I0103ace2afda2bf457ff45a05cc7245dfed8d6bb
---

diff --git a/scripts/serdes/gaudi3_serdes_config_flash_memory.json b/scripts/serdes/gaudi3_serdes_config_flash_memory.json
index 022cba8..699e6b2 100644
--- a/scripts/serdes/gaudi3_serdes_config_flash_memory.json
+++ b/scripts/serdes/gaudi3_serdes_config_flash_memory.json
@@ -1,6 +1,7 @@
 {
 	"serdes_table": [
 		{
+			"name": "HLS3_UNKNOWN_SERDES_LINK_ID",
 			"type": 6,
 			"link_id": "0b00001",
 			"is_node_exists" : false,
@@ -75,6 +76,7 @@
 			]
 		},
 		{
+			"name": "HLS3_FULLSCALE_IN_SERDES_LINK_ID",
 			"type": 6,
 			"link_id": "0b00100",
 			"is_node_exists" : false,
@@ -149,6 +151,7 @@
 			]
 		},
 		{
+			"name": "HLS3_FULLSCALE_OUT_SERDES_LINK_ID",
 			"type": 7,
 			"link_id": "0b00010",
 			"is_node_exists" : false,
@@ -222,6 +225,7 @@
 			]
 		},
 		{
+			"name": "HLS3_FULL_OAM_3PORTS_SCALE_OUT_SERDES_LINK_ID",
 			"type": 8,
 			"link_id": "0b11110",
 			"is_node_exists" : false,
@@ -295,6 +299,7 @@
 			]
 		},
 		{
+			"name": "HLS3_FULL_OAM_6PORTS_SCALE_OUT_SERDES_LINK_ID",
 			"type": 9,
 			"link_id": "0b10000",
 			"is_node_exists" : false,
@@ -368,6 +373,7 @@
 			]
 		},
 		{
+			"name": "HLS3_SINGLEPORT_OAM_FULLSCALE_OUT_SERDES_LINK_ID",
 			"type": 10,
 			"link_id": "0b00000",
 			"is_node_exists" : false,
@@ -441,6 +447,7 @@
 			]
 		},
 		{
+			"name": "HLB325_FULL_OAM_3PORTS_SCALE_OUT_SERDES_LINK_ID",
 			"type": 12,
 			"link_id": "0b11111",
 			"is_node_exists" : false,
@@ -514,6 +521,7 @@
 			]
 		},
 		{
+			"name": "GAUDI3_RACK_SKYRIVER3_SERDES_LINK_ID",
 			"type": 15,
 			"link_id": "0b01000",
 			"is_node_exists" : true,
@@ -556,6 +564,7 @@
 			]
 		},
 		{
+			"name": "GAUDI3_RACK_HUAQIN_WHITEBOX_SERDES_LINK_ID",
 			"type": 16,
 			"link_id": "0b01100",
 			"is_node_exists" : true,
@@ -598,4 +607,4 @@
 			]
 		}
 	]
-}
\ No newline at end of file
+}
